src/js/index.js

```diff
@@ -9,92 +11,437 @@ const deepFreeze = obj => {
   return Object.freeze(obj);
 };
 
-const accentColors = ['#2AD2C9', '#614767', '#ff8d6d'];
-const neutralColors = ['#425563', '#5F7A76', '#80746E', '#767676'];
-const statusColors = {
-  critical: '#F04953',
-  error: '#F04953',
-  warning: '#FFD144',
-  ok: '#01a982',
-  unknown: '#CCCCCC',
-  disabled: '#CCCCCC',
-};
-
-const colors = {
-  brand: '#01a982',
-  focus: accentColors[0],
-};
-
-const colorArray = (array, prefix) =>
-  array.forEach((color, index) => {
-    colors[`${prefix}-${index + 1}`] = color;
-  });
-
-colorArray(accentColors, 'accent');
-colorArray(neutralColors, 'neutral');
-Object.keys(statusColors).forEach(color => {
-  colors[`status-${color}`] = statusColors[color];
-});
-
 export const hpe = deepFreeze({
+  // name: 'HPE 2',
```

Comment:
Why are we commenting these out?

Reply:
I commented-out attributes that don't have actual impact on the code. So we wouldn't imply this attribute has another meaning of just the name.

Reply:
It means the HPE theme will be inconsistent with themes coming from the theme designer then, anyone who builds a theme with the theme designer will have these properties set. And, I don't think that's a problem, it allows the theme to be imported/pasted into the theme designer and then modified.